src/components/visualizations/chart/highcharts/plugins/adjustTickAmount.ts

```diff
@@ -240,32 +276,29 @@ function isAxisWithLineChartType(axis: IHighchartsAxisExtend): boolean {
         return true;
     }
 
-    const { series } = axis;
+    const { series = [] } = axis;
```

Comment:
the same with previous comment, could `series` be null somehow?

Reply:
done